src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,33 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Add your docs here.
+ */
+public class Drivetrain extends Subsystem {
+  public static void manualcontrol (TalonSRX port, TalonSRX starboard, Joystick stick){
+    Double thro = stick.getRawAxis(1);
+    Double yaw = stick.getRawAxis(2);
+    port.set(ControlMode.PercentOutput, thro);
+    starboard.set(ControlMode.PercentOutput, (thro - yaw));
```

Comment:
The motor on this side faces backwards, make sure to reverse it by multiplying it by -1
starboard.set(ControlMode.PercentOutput, (thro * -1) - yaw);